### ISSUE TITLE
Duplicates in showcase

### ISSUE THREAD

**Issue by yorkshire-pudding** [backdrop-ops/backdropcms.org#892]
There are currently quite a few sites in the showcase that are shown twice, sometimes next to each other, sometimes spread apart.  Please can someone with admin access to the views look at this.  This is two just on the first page:
![image](https://user-images.githubusercontent.com/20357797/158355539-9bc74084-daf1-4651-ad34-7e11c1b46e69.png)

**Comment by BWPanda** [comment]
It looks like the addition of the homepage block added a filter that was causing duplicates, since it was applied globally and not just to the new block display. I've fixed that so this should be all good now.

@jenlampton I assume you were the one making these views changes (since you seemed active in the issue where this was being discussed). I didn't see any recent PRs that applied the changes, so I assume you just made them directly on the site. I therefore assumed it was ok that I did the same.
